### PR TITLE
Feature/menu

### PR DESCRIPTION
**Pull request by mluena**
- WIP menu

### CODE REVIEW

File: components/home/header-desktop/index.js [hunk 1]
```diff
@@ -1,18 +1,108 @@
...
+      </p>
+
+      <button className="header-menu--button" onClick={handleMenu}>
+        <Icon name={isOpen ? "menu-close" : "menu"} className="-medium -dark" />
+      </button>
+      {isOpen && (
+        <div className="menu-content" role="navigation">
+          <ul className="menu" role="menubar">
+            <li className="menu-item">
+              <h2 className="-block0" role="menuitem" tabIndex="0">Building knowledge.</h2>
```
Comment (Bluesmile82) [inline components/home/header-desktop/index.js]: I think we should remove the role menuitem here as it is not selectable

File: components/home/header-desktop/index.js [hunk 2]
```diff
@@ -1,18 +1,108 @@
...
-);
+const HomeHeaderDesktop = () => {
+  const [isOpen, setMenu] = useState(false);
+
+  const handleMenu = () => {
+    setMenu(!isOpen);
+  };
+
+  return (
+    <div className={cx('c-header-desktop', { '-open': isOpen })}>
```
Comment (Bluesmile82) [inline components/home/header-desktop/index.js]: Need to add role="menu" aria-labelledby="main-menu-button" id="main-menu"

Also the button should have id="main-menu-button" aria-haspopup="true" aria-controls="main-menu" aria-expanded="true" (aria expanded true when opened and false when closed)

https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

File: components/home/header-mobile/index.js [hunk 3]
```diff
@@ -13,27 +15,83 @@ const HomeHeaderMobile = () => {
...
-            target="_blank"
-            className="privacy-link menu-item"
-          >
-            Privacy policy.
-          </a>
+          <ul className="menu">
+            <li className="menu-item">
+              <h2 className="-block0">Building knowledge.</h2>
+              <div>
+                <Link href={{ pathname: '/report/0/[slug]', query: { slug: '0' } }}>
```
Comment (Bluesmile82) [inline components/home/header-mobile/index.js]: This won't work as the rendered page on the link its not exactly the same as the one rendered by the modal. You can use a router.push as we have on the pieces or we can save the progress in sessionStorage or similar

File: components/home/header-desktop/index.js (hunk 1 above)
Comment (mluena) [reply components/home/header-desktop/index.js]: I left it with a tabIndex='-1' y also added and id and an aria-labelledby to that id for the ul contained in that li

File: components/home/header-mobile/index.js [hunk 4]
```diff
@@ -136,13 +153,31 @@ const HomeHeaderMobile = ({
               className="privacy-link sub-menu-item"
             >
               Privacy policy.
-            </a>
+           </a>
           </div>
-
-        </div>
+          <SoundButton
+            className="-absolute -center hola"
```
Comment (Bluesmile82) [inline components/home/header-mobile/index.js]: 👋

File: components/home/header-desktop/styles.scss [hunk 5]
```diff
@@ -7,6 +7,21 @@
   &.-open {
     background-color: black;
     height: 100%;
+
+    &:before {
```
Comment (Bluesmile82) [inline components/home/header-desktop/styles.scss:12]: maybe we can share this styles with a mixin

**Pull request merged**